src/integrationTest/resources/envelope.json

```diff
@@ -20,7 +20,7 @@
           "file_name": "1111002.pdf",
           "notes": "test",
           "document_url": "http://localhost:8080/documents/0fa1ab60-f836-43aa-8c65-b07cc9bebcbe",
-          "document_type": "Other"
+          "document_type": "OTHER"
```

Comment:
Why is this one all capital?

Reply:
This one is used in test for API response. Responses are directly converted from enum using `.toString()` which is capital. Therefore note in the description

Reply:
I see. Is the contract agreed somehow? If not, probably doesn't matter, but if it is, we could use a custom JSON serialiser for this enum.

Reply:
I think we are not using this service as REST at all. Previously there was a plan but now everything got moved to ServiceBus. So most likely all web related stuff is redundant. Perhaps can be reviewed in next meeting